sustainability/src/main/java/edu/asu/diging/sustainability/core/model/impl/Concept.java

```diff
@@ -82,6 +85,39 @@ public void setParent(IConcept parent) {
 
     @Override
     public List<IConcept> getChildren() {
+        
```

Comment:
this is for a very special case only (for SDGs), so you should check that you are sorting only the SDGs by numbers, otherwise this could have (even if not at the moment) unintended side effects later on (e.g. sorting My1Concept and Test2 by numbers).